README.md

```diff
@@ -1,69 +1,185 @@
-# decomp.me
-A collaborative decompilation and reverse engineering platform
+# [decomp.me](https://decomp.me)
 
-## Contributing
+A collaborative decompilation and reverse engineering website, built with React and Django.
 
-### Directory structure
-`frontend/` contains the website sourcecode
+## Features
 
-`backend/` contains the Django project
+🐸 Create 'scratches' to share and work on function decompilations
```

Comment:
I missed these frogs lol. I like them but I think they're a tad too silly for the readme

I'm not even sure we need the features section - seems like another thing to maintain when we add more compilers and stuff